backends/advanced/src/advanced_omi_backend/controllers/system_controller.py

```
@@ -284,159 +284,100 @@ async def get_speaker_service_status():
+        config_yaml = yaml.safe_dump(memory_section, sort_keys=False)
+
         return {
             "config_yaml": config_yaml,
-            "config_path": config_path,
-            "status": "success"
+            "config_path": str(cfg_path),
+            "section": "memory",
+            "status": "success",
         }
-        
     except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)